arangod/VocBase/Methods/Collections.cpp

```diff
@@ -602,10 +602,12 @@ Result Collections::properties(Context& ctxt, VPackBuilder& builder) {
 }
 
 Result Collections::updateProperties(LogicalCollection& collection,
-                                     velocypack::Slice const& props, bool partialUpdate) {
+                                     velocypack::Slice const& props) {
+  const bool partialUpdate = false; // always a full update
```

Comment:
it looks like `partialUpdate` is passed as a parameter to `LogicalCollection::properties()`, but there it does not seem to be used at all. So can we remove this variable and remove it from the signature of `LogicalCollection::properties()`?

Reply:
it is used in Views which inherit this method from logicaldatasource

Reply:
oh, ok.